scripts/jenkins/build_docker.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -euxo pipefail
+
+if ! which docker
+then
+  echo "ERROR: Building Docker image requires Docker binary to be installed" && exit 1
+elif ! docker version
+then
+  echo "ERROR: Building Docker image requires Docker daemon to be running" && exit 1
+elif ! which curl
+then
+  echo "ERROR: Building Docker image requires cURL to be installed" && exit 1
+fi
+
+echo "INFO: Fetching latest tag"
+#TODO Move this into apm-pipeline-library
+readonly GIT_TAG=`git describe --abbrev=0|sed s/^v//`
```

Comment:
Allow passing the GIT_TAG on the environment, by default use the latest TAG

```suggestion
GIT_TAG_DEFAULT=$(git describe --abbrev=0|sed s/^v//)
readonly GIT_TAG=${GIT_TAG:-$GIT_TAG_DEFAULT}
```